src/__tests__/next-component.test.tsx

```
@@ -0,0 +1,252 @@
+import { render, screen } from "@testing-library/react";
+
+// Mock Next.js modules
+vi.mock("next/script", () => ({
+  default: (props: Record<string, unknown>) => (
+    <script data-testid="next-script" {...props} />
+  ),
+}));
+
+vi.mock("next/image", () => ({
+  default: (props: Record<string, unknown>) => (
+    // eslint-disable-next-line @next/next/no-img-element, jsx-a11y/alt-text
```

ESLint

[error] Definition for rule '@next/next/no-img-element' was not found.

(`@next/next/no-img-element`)

---

[error] Definition for rule 'jsx-a11y/alt-text' was not found.

(jsx-a11y/alt-text)